utils/CreateDataset.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, work_folder,
                 metadata = hdf5_file["metadata"][i, ...]
                 if isinstance(metadata, np.ndarray) and len(metadata) == 1:
                     metadata = metadata[0]
+                    metadata = metadata.decode('ascii')
```

Comment:
@victorlazio109 @remtav can you look at this i am not familiar with this file.

Reply:
@mpelchat04 and @felegare have worked more than @victorlazio109 and I with metdata. Any input?
@mkutu I'm guessing this line was necessary for your use of metadata?

Reply:
yes, this was necessary for me. Perhaps I should make another issue for this? 
I was getting, AttributeError: 'bytes' object has no attribute 'update'

I could not begin training because the hdf5 file was being read as non-unicode string which python displays as bytes ('b').